content/geoip/release-notes/2024.mdx

```diff
@@ -7,6 +7,31 @@ title: GeoIP2 Release Notes
 Subscribe to the [GeoIP2 release notes RSS feed](/geoip/release-notes/rss.xml).
 </Alert>
 
+<ReleaseNote date="2024-07-08" title="GeoNames Monthly Diff Report August 2024">
```

Comment:
Is this the right date?

Reply:
ah no, thank you for catching that! I've just done a push for the correct date.